cli/pcluster/config/validators.py

```diff
@@ -1200,3 +1209,17 @@ def fsx_ignored_parameters_validator(section_key, section_label, pcluster_config
                 errors.append(FSX_MESSAGES["errors"]["ignored_param_with_fsx_fs_id"].format(fsx_param=fsx_param))
 
     return errors, warnings
+
+
+def queue_xor_cluster_params_validator(param_key, param_value, pcluster_config):
+    """Return error if the same param is specified in the cluster section."""
+    errors = []
+    cluster_section_value = pcluster_config.get_section("cluster").get_param_value(param_key)
+    if param_value is not None:
+        if cluster_section_value is not None:
+            errors.append("Param '{0}' can be used only in 'cluster' or in 'queue' section".format(param_key))
```

Comment:
I'd write: `"Parameter 'xxx' can be used in only one of the '[cluster ...]' or '[queue ...]' sections`. 
 @nstornetta do you have a better suggestion?

Reply:
I wonder "only one" could be interpreted as "only in one queue section". Instead, if it is not specified in ``cluster`` section, it must be specified in *all* `queue` sections.

Reply:
OK for Param -> Parameter